### PR TITLE
replace plant photo placeholder

### PR DESCRIPTION
**Pull request by marynag**

### CODE REVIEW

**Review by Nazar-kob — APPROVED** [review]

File: src/components/PlantCardPreview/PlantCardPreview.tsx [hunk 1]
```diff
@@ -46,7 +41,7 @@ export const PlantCardPreview = ({ plantInfo }: PlantCardPreviewProps) => {
 			<FastImage
 				style={styles.plantImage}
 				source={{
-					uri: photo?.urlPath || getPlantImagePlaceholder(),
+					uri: photo?.urlPath || Image.resolveAssetSource(placeholderImage).uri,
```
Comment (Taraslysak) [inline src/components/PlantCardPreview/PlantCardPreview.tsx:44]: I think we can get this uri only once, outside of component function scope, so as not to load this image on each re-render

File: src/components/PlantPhotoSwiper/PlantPhotoSwiper.tsx [hunk 2]
```diff
@@ -37,32 +34,24 @@ export const PlantPhotoSwiper = () => {
...
-								resizeMode={FastImage.resizeMode.cover}
-							/>
-					  ))
-					: DEFAULT_PHOTOS.map((photo) => (
-							<FastImage
-								key={photo.uri}
-								style={styles.plantImage}
-								source={{
-									uri: photo.uri,
+									uri: photo?.urlPath || Image.resolveAssetSource(placeholderImage).uri,
```
Comment (Taraslysak) [inline src/components/PlantPhotoSwiper/PlantPhotoSwiper.tsx:46]: Same as above

File: src/components/PlantPhotoSwiper/PlantPhotoSwiper.tsx [hunk 3]
```diff
@@ -37,32 +34,24 @@ export const PlantPhotoSwiper = () => {
 	}
 
 	return (
-		<View style={styles.wrapper}>
-			<Swiper loop activeDotColor={styles.activeDot.backgroundColor} width={width}>
-				{data?.data.length && !!data.data.length
-					? data.data.map((photo) => (
+		<>
```
Comment (Taraslysak) [inline src/components/PlantPhotoSwiper/PlantPhotoSwiper.tsx:37]: Do we have story for this compoent?

If no, why not?

**Review by Taraslysak — COMMENTED** [review]
Please consider some refactoring

**Pull request merged**